### ISSUE TITLE
setSelectionDrawable(Drawable drawable) not working for me

### ISSUE THREAD

**Issue by szykam** [prolificinteractive/material-calendarview#72]
Hi,
What I want is to mark selected date with drawable from resources (PNG file). I took BasicActivity from sample application and added this code:

        DayViewDecorator decorator = new DayViewDecorator() {
            @Override
            public boolean shouldDecorate(CalendarDay day) {
                return true;
            }

            @Override
            public void decorate(DayViewFacade view) {
                if(Build.VERSION.SDK_INT >= Build.VERSION_CODES.LOLLIPOP) {
                    view.setSelectionDrawable(getDrawable(R.drawable.select_date));
                } else {
                    view.setSelectionDrawable(ContextCompat.getDrawable(BasicActivity.this, R.drawable.select_date));
                }
            }
        };
        widget.addDecorator(decorator);

Unfortunately when I select a date only default, blue, circle background is displayed. Am I missing something?

**Comment by dandc87** [comment]
Hi @szykam, the selection drawable needs to be a [state list drawable](http://developer.android.com/guide/topics/resources/drawable-resource.html#StateList). You would need to create an xml drawable containing something like this:

```xml
<?xml version="1.0" encoding="utf-8"?>
<selector xmlns:android="http://schemas.android.com/apk/res/android">
    <item
        android:state_checked="true"
        android:drawable="@drawable/select_date"
        />
    <item
        android:state_pressed="true"
        android:drawable="@drawable/select_date"
        />
    <item android:drawable="@android:color/transparent" />
</selector>
```

I'll make sure to add documentation about how to customize stateful things.

**Comment by szykam** [comment]
Thanks for an answer, but sadly this solution doesn't work form me. Well it works but with additional tweaks.
1. Calling DayViewFacade#setSelectionDrawable(Drawable drawable) doesn't work. DayViewFacade#setBackgroundDrawable(Drawable drawable) must be called.

2. Although my PNG from a selector is drawn for selected day, color circle background is also drawn.

3. Calling MaterialCalendarView#setSelectionColor(int color) with transparent color makes no effect. To get rid of circle background I've passed almost transparent color (#01000000).

**Comment by dandc87** [comment]
@szykam Fixed in 0.7.0
